backend_new/package.json

```diff
@@ -16,7 +16,7 @@
     "lint": "eslint \"{src,apps,libs,test}/**/*.ts\" --fix",
     "test": "jest --config ./test/jest.config.js",
     "test:watch": "jest --watch",
-    "test:cov": "yarn db:resetup && yarn db:migration:run && jest --config ./test/jest-with-coverage.config.js",
```

Comment:
I think dropping `db:resetup` creates some friction when running the suite locally

is it possible if we do something like:
```
"test:cov": "yarn db:resetup && db:migration:run && jest --config ./test/jest-with-coverage.config.js --logHeapUsage",
"test:cov-ci": "yarn db:migration:run && jest --config ./test/jest-with-coverage.config.js --runInBand --logHeapUsage",
```

so we can maintain running cov locally and create a spot for ci to operate?